### PR TITLE
enh(phpcs) : upgrade phpcs version

### PR DESCRIPTION
**Pull request by AssanDiomande**
## Description

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x
- [x] 22.04.x (master)

### CODE REVIEW

File: www/class/centreonLog.class.php [hunk 1]
```diff
@@ -131,7 +131,6 @@ public static function singleton($uid = 0)
 
 class CentreonLog
```
[the remark this answers is not in the document]
Comment (AssanDiomande) [reply www/class/centreonLog.class.php:132]: to be ignored

**Review by kduret — DISMISSED** [review]

File: src/Centreon/Domain/Entity/Options.php [hunk 2]
```diff
@@ -4,25 +4,45 @@
...
     private $key;
+
+    /**
+     * @var string
+     */
     private $value;
 
+    /**
+     * @param string $key
+     * @return void
```
Comment (adr-mo) [inline src/Centreon/Domain/Entity/Options.php:19]: I guess this is not necessary
```suggestion
```

File: src/Centreon/Domain/Entity/Options.php [hunk 3]
```diff
@@ -4,25 +4,45 @@
...
+     * @return string
+     */
     public function getKey(): string
     {
         return $this->key;
     }
 
+    /**
+     * @param string $value
+     * @return void
```
Comment (adr-mo) [inline src/Centreon/Domain/Entity/Options.php:36]:
```suggestion
```

File: src/Centreon/Tests/Application/DataRepresenter/BulkTest.php [hunk 4]
```diff
@@ -42,8 +42,10 @@
 
 class BulkTest extends TestCase
 {
-
-    public function testJsonSerialize()
+    /**
+     * @return void
+     */
```
Comment (adr-mo) [inline src/Centreon/Tests/Application/DataRepresenter/BulkTest.php:47]: docBloc should describe what the test does
```suggestion
```

File: src/Centreon/Tests/Application/DataRepresenter/EntityTest.php [hunk 5]
```diff
@@ -41,8 +41,10 @@
 
 class EntityTest extends TestCase
 {
-
-    public function testJsonSerialize()
+    /**
+     * @return void
+     */
```
Comment (adr-mo) [inline src/Centreon/Tests/Application/DataRepresenter/EntityTest.php:46]: same

File: src/Centreon/Tests/Domain/Repository/Traits/CheckListOfIdsTraitTest.php [hunk 6]
```diff
@@ -44,17 +44,24 @@
 
 class CheckListOfIdsTraitTest extends TestCase
 {
-
-    public function testCheckListOfIdsTrait()
+    /**
+     * @return void
+     */
```
Comment (adr-mo) [inline src/Centreon/Tests/Domain/Repository/Traits/CheckListOfIdsTraitTest.php:49]: same

File: src/Centreon/Domain/Entity/Options.php (hunk 2 above)
Comment (AssanDiomande) [reply src/Centreon/Domain/Entity/Options.php:19]: done

File: src/Centreon/Domain/Entity/Options.php (hunk 3 above)
Comment (AssanDiomande) [reply src/Centreon/Domain/Entity/Options.php:36]: done

File: src/Centreon/Tests/Application/DataRepresenter/BulkTest.php (hunk 4 above)
Comment (AssanDiomande) [reply src/Centreon/Tests/Application/DataRepresenter/BulkTest.php:47]: done

File: src/Centreon/Tests/Domain/Repository/Traits/CheckListOfIdsTraitTest.php (hunk 6 above)
Comment (AssanDiomande) [reply src/Centreon/Tests/Domain/Repository/Traits/CheckListOfIdsTraitTest.php:49]: done

File: src/Centreon/Tests/Application/DataRepresenter/EntityTest.php (hunk 5 above)
Comment (AssanDiomande) [reply src/Centreon/Tests/Application/DataRepresenter/EntityTest.php:46]: done

**Review by kduret — APPROVED** [review]

**Review by adr-mo — APPROVED** [review]

**Pull request merged**